### ISSUE TITLE
Roadmap of MMSegmentation

### ISSUE THREAD

**Issue by hellock** [open-mmlab/mmsegmentation#13]
We keep this issue open to collect feature requests from users and hear your voice. Our monthly release plan is also available here.

You can either:
1. Suggest a new feature by leaving a comment.
2. Vote for a feature request with 👍 or be against with 👎. (Remember that developers are busy and cannot respond to all feature requests, so vote for your most favorable one!)
3. Tell us that you would like to help implement one of the features in the list or review the PRs. (This is the greatest things to hear about!)

**Comment by xvjiarui** [comment]
#18

**Comment by xvjiarui** [comment]
albumentation in #29

**Comment by zjupx** [comment]
MobileNet v3 backbone support please！

**Comment by VincentZ-2020** [comment]
Add U-net please！

**Comment by zhongqiu1245** [comment]
nnU-net support please！

**Comment by xungeer29** [comment]
tensorboard support please!

**Comment by xvjiarui** [comment]
> tensorboard support please!

You may uncomment this [line](https://github.com/open-mmlab/mmsegmentation/blob/master/configs/_base_/default_runtime.py#L6) to enable tensorboard.

**Comment by JoyLuo** [comment]
Gans for segment

**Comment by advaza** [comment]
Is there a way to view segmentation results throughout training (like evaluation hook but with visualisation of possibly unlabelled dataset)? 
E.g. Tensorboard image writer.

**Comment by sandylaker** [comment]
UNet, UNet++, and installation on OS X.

**Comment by hhaAndroid** [comment]
support  EfficientFCN

**Comment by JoyLuo** [comment]
BCE loss， focal loss

**Comment by caodroid** [comment]
please add Res2net，thank you，

**Comment by jianlong-yuan** [comment]
please add Panoptic-DeepLab and segfix, thank you

**Comment by huang45** [comment]
https://github.com/PingoLH/FCHarDNet please!

**Comment by huang45** [comment]
and Bisenetv2, thank you~

**Comment by Hr-Song** [comment]
**Lovasz hinge loss** and **Lovasz softmax** loss suport please， thank you.

**Comment by ywwwer** [comment]
Multi-scale attention [https://github.com/NVIDIA/semantic-segmentation](url), thank you!

**Comment by zhongqiu1245** [comment]
HighRes2DNet(2D version of HighRes3DNet) support please!

**Comment by zhongqiu1245** [comment]
support CaNet please，thank you！
https://github.com/icoz69/CaNet?utm_source=wechat_session&utm_medium=social&utm_oi=980147122032807936

**Comment by caodroid** [comment]
hope the Xception will be supported in backbones, thank you []https://arxiv.org/pdf/1610.02357.pdf(url)

**Comment by JPLAY0** [comment]
Hope add resnet18 pretrained model ,thank you.

**Comment by 9p15p** [comment]
support COCO Dataset, please. Thank you.

**Comment by JPLAY0** [comment]
Support Mapillary Vistas Dataset ,thank you.

**Comment by zhongqiu1245** [comment]
Support Dice Loss & Focal Loss, thankyou!

**Comment by NickChang97** [comment]
Make it easy to use mixed loss

**Comment by lucci17** [comment]
please support for temporal structure

**Comment by caodroid** [comment]
DenseASPP plz，thank you

**Comment by JPLAY0** [comment]
Support CamVid Dataset ,thank you.

**Comment by jfzhuang** [comment]
Xception backbone support please！

**Comment by fcakyon** [comment]
Please support [VNet](https://arxiv.org/abs/1606.04797) and [ResUNet](https://arxiv.org/abs/1904.00592)

**Comment by aikeyz** [comment]
EMA/SWA and knowledge distillation please

**Comment by zhongqiu1245** [comment]
Hi, thanks for your amazing job！
Is there any plan to support Panoptic-DeepLab？
Thank you！

**Comment by hannes09** [comment]
Hi, 
great project. I would like to see DDRNet.
https://github.com/chenjun2hao/DDRNet.pytorch

**Comment by tiwaria9034** [comment]
Thanks for your contributions.
Please add EfficientNet family of backbones (v1, v2, tflite, etc.)

**Comment by Junjun2016** [comment]
> Thanks for your contributions.
> Please add EfficientNet family of backbones (v1, v2, tflite, etc.)

Could you please provide some references?

**Comment by tiwaria9034** [comment]
> > Thanks for your contributions.
> > Please add EfficientNet family of backbones (v1, v2, tflite, etc.)
> 
> Could you please provide some references?

EfficientNet (v1): 
- https://arxiv.org/abs/1905.11946
- https://github.com/rwightman/pytorch-image-models/blob/master/timm/models/efficientnet.py

EfficientNet (v2): 
- https://arxiv.org/pdf/2104.00298.pdf
- https://github.com/google/automl/tree/master/efficientnetv2

**Comment by 8639sung** [comment]
Please support Ensemble! or at least, soft label output for ensemble

**Comment by eliabruni** [comment]
Provide TensorRT export for SegFormer (as a very embed- and mobile-friendly transformer-based segmentor)

**Comment by eliabruni** [comment]
Hi, great project. 
I would like to see [Mask2Former](https://github.com/facebookresearch/Mask2Former)

**Comment by wmkai** [comment]
Hi, thanks for your great job！
Is there any plan to support Mask2Former？
Thank you！

**Comment by jyang68sh** [comment]
Please update PIDnet as it is the new sota

**Comment by FabianSchuetze** [comment]
Could we add `SegNext`? 

Code: https://github.com/visual-attention-network/segnext
paper: https://arxiv.org/abs/2209.08575

The code already uses mmsegmentation.

**Comment by joihn** [comment]
Thanks for this repo,
Any plan to support RtFormer ? 
https://arxiv.org/abs/2210.07124
